types/htmlbars-inline-precompile/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for htmlbars-inline-precompile 1.0
+// Project: ember-cli-htmlbars-inline-precompile
```

Comment:
please add a url to docs, e.g.: https://github.com/ember-cli/ember-cli-htmlbars-inline-precompile